### PR TITLE
Add endpoints for FID<->internal_id by type for k8s

### PR DESCRIPTION
**Pull request by abhishekdharmapurikar**
Add endpoints for FID<->internal_id by type for k8s

### CODE REVIEW

File: pkg/polaris/graphql/infinity-k8s/infinity_k8s.go [hunk 1]
```diff
@@ -229,6 +229,19 @@ func Wrap(client *polaris.Client) API {
 	return API{GQL: client.GQL, log: client.GQL.Log()}
 }
 
+// K8sObjectType is the type of the Kubernetes object. One between SLA,
+// INVENTORY, and SNAPSHOTS.
```
Comment (sai-sandeep-1) [inline pkg/polaris/graphql/infinity-k8s/infinity_k8s.go]: Nit: K8sObjectType is the type of Kubernetes object that can be one of SLA, INVENTORY, or SNAPSHOTS.

File: pkg/polaris/graphql/infinity-k8s/infinity_k8s_test.go [hunk 2]
```diff
@@ -400,3 +400,88 @@ func TestIntegrationTemp(t *testing.T) {
...
 	logger.Printf(log.Info, "Assign SLA response %v\n", slaResp)
 }
+
+func TestIntegrationTranslation(t *testing.T) {
+	ctx := context.Background()
+
+	// if !testsetup.BoolEnvSet("TEST_INTEGRATION") {
+	// 	t.Skipf("skipping due to env TEST_INTEGRATION not set")
+	// }
+
```
Comment (sai-sandeep-1) [inline pkg/polaris/graphql/infinity-k8s/infinity_k8s_test.go]: Do you need to uncomment this?

File: pkg/polaris/graphql/infinity-k8s/queries/k8s_object_fid_by_type.graphql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+query RubrikPolarisSDKRequest($k8SObjectInternalIdArg: UUID!, $clusterUuid: UUID!, $kubernetesObjectType: kubernetesObjectType!) {
+    k8sObjectFidByType(clusterUuid: $clusterUuid, K8sObjectInternalIDArg: $k8SObjectInternalIdArg, kubernetesObjectType: $kubernetesObjectType)
```
Comment (sai-sandeep-1) [inline pkg/polaris/graphql/infinity-k8s/queries/k8s_object_fid_by_type.graphql:2]: Are these newly introduced GQLs?

**Review by sai-sandeep-1 — APPROVED** [review]
Left a few comments, PTAL. Approving to unblock

**Review by pinzhourk — APPROVED** [review]

**Review by rajath26 — APPROVED** [review]

File: pkg/polaris/graphql/infinity-k8s/infinity_k8s_test.go (hunk 2 above)
Comment (abhishekdharmapurikar) [reply pkg/polaris/graphql/infinity-k8s/infinity_k8s_test.go]: Oh will do that.

File: pkg/polaris/graphql/infinity-k8s/queries/k8s_object_fid_by_type.graphql (hunk 3 above)
Comment (abhishekdharmapurikar) [reply pkg/polaris/graphql/infinity-k8s/queries/k8s_object_fid_by_type.graphql:2]: Yes new ones.

File: pkg/polaris/graphql/infinity-k8s/infinity_k8s.go (hunk 1 above)
Comment (abhishekdharmapurikar) [reply pkg/polaris/graphql/infinity-k8s/infinity_k8s.go]: Will fix.

**Pull request merged**